<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PassGenerator</class>
 <widget class="QDialog" name="PassGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="kol_sim">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>140</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Кол-во символов:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="kol_pass">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>100</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="pass_spis">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>211</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Пароли</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Кол-во паролей</string>
   </property>
  </widget>
  <widget class="QLabel" name="secret_text">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>180</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Желаемое исходное слово</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="rec_slova">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="gen_knopka">
   <property name="geometry">
    <rect>
     <x>439</x>
     <y>314</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать пароли</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sohr_knopka">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить в .txt</string>
   </property>
  </widget>
  <widget class="QLabel" name="ent_text">
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>256</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
